tshirts.c

```diff
@@ -14,9 +14,10 @@ char size(int cms) {
 }
 
 int main() {
-    assert(size(37) == 'S');
-    assert(size(40) == 'M');
-    assert(size(43) == 'L');
+    assert(size(38) == 'S');

```

Comment:
Test cases need to be strengthened here to cover all boundary conditions and values before and after.

Reply:
i will add them